system/lib/wasmfs/syscalls.cpp

```diff
@@ -1239,7 +1239,7 @@ int __syscall_ioctl(int fd, int request, ...) {
       return -EINVAL;
     }
     default: {
-      abort();
+      return -EINVAL; // not supported
```

Comment:
Combine with the abort block?  Actually better still just remove `TIOCGPGRP`/`TIOCSPGRP` above